globals.go

```diff
@@ -206,8 +206,9 @@ const (
 	FinderVersionPipeline  = "pipeline"
 	FinderVersionAlternate = "alternate"
 
-	HostAllocatorDeficit      = "deficit"
-	HostAllocatorUtilization  = "utilization"
+	HostAllocatorDeficit     = "deficit"
+	HostAllocatorUtilization = "utilization"
+
```

Comment:
(these are different groupings which I neglected to separate when I first wrote them)